helpdesk-bot/src/test/java/org/symphonyoss/symphony/bots/helpdesk/bot/it/steps/TicketSteps.java

```diff
@@ -0,0 +1,48 @@
+package org.symphonyoss.symphony.bots.helpdesk.bot.it.steps;
+
+import org.jbehave.core.annotations.Then;
+import org.springframework.stereotype.Component;
+import org.symphonyoss.client.SymphonyClient;
+import org.symphonyoss.client.exceptions.MessagesException;
+import org.symphonyoss.client.exceptions.StreamsException;
+import org.symphonyoss.symphony.bots.helpdesk.bot.it.TestContext;
+import org.symphonyoss.symphony.bots.helpdesk.bot.it.exception.UserNotAuthenticatedException;
+import org.symphonyoss.symphony.clients.model.SymMessage;
+import org.symphonyoss.symphony.clients.model.SymStream;
+import org.symphonyoss.symphony.clients.model.SymUser;
+
+/**
+ * Class responsible for managing ticket steps.
+ * <p>
+ * Created by rsanchez on 23/02/18.
+ */
+@Component
```

Comment:
Is the annotation required?

Reply:
Yes, it's required to support dependency injection of application Symphony client and then get the bot user info.